tests/safeds_stubgen/stubs_generator/__snapshots__/test_generate_stubs/test_stub_docstring_creation[docstring_vs_typehints-THROW_WARNING].sdsstub

```diff
@@ -0,0 +1,64 @@
+@PythonModule("tests.data.docstring_parser_package.docstring_vs_typehints")
+package tests.data.docstringParserPackage.docstringVsTypehints
+
+/**
+ * function_with_result_value_and_type.
+ *
+ * Dolor sit amet.
+ *
+ * @result namedResult this will be the return value
+ */
+@Pure
+@PythonName("doc_vs_hints_function_with_parameters_and_return")
+fun docVsHintsFunctionWithParametersAndReturn(
+    x: Int = 1,
+    y: String = "Some String"
+) -> namedResult: Boolean
+
+/**
+ * function_with_result_value_and_type.
+ *
+ * Dolor sit amet.
+ *
+ * @result namedResult this will be the return value
+ */
+@Pure
+@PythonName("doc_vs_hints_function_with_contrasting_parameter_and_return")
+fun docVsHintsFunctionWithContrastingParameterAndReturn(
+    x: String = "Some String"
+) -> namedResult: Int
+
+/**
+ * function_with_result_value_and_type.
+ *
+ * Dolor sit amet.
+ *
+ * @result firstResult first float value instead of int
+ * @result secondResult second bool value instead of str
+ */
+@Pure
+@PythonName("doc_vs_hints_function_with_multiple_contrasting_return")
+fun docVsHintsFunctionWithMultipleContrastingReturn() -> (result1: Int, result2: String)
```

Comment:
Wrong result names, probably due to https://github.com/Safe-DS/Stub-Generator/issues/138.